buildenv/jenkins/JenkinsfileBase

```diff
@@ -12,6 +12,11 @@ def test() {
 				env.JAVA_HOME = "${JAVA_BIN}/.."
 				env.JAVA_VERSION = "${JAVA_VERSION}"
 				env.JVM_VERSION = "${JVM_VERSION}"
+				try{
+					env.CUSTOMIZED_SDK_URL = "${CUSTOMIZED_SDK_URL}"
+				} catch (MissingPropertyException e) {
+					env.CUSTOMIZED_SDK_URL = ''
+				}
```

Comment:
CUSTOMIZED_SDK_URL is only used for get.sh. A simple parameter should be enough. Is there any reason to be a environment variable?

Reply:
The only reason CUSTOMIZED_SDK_URL was made an env variable is because all the other variables are env variables too (e.g. JAVA_VERSION, JVM_VERSION etc).

Reply:
I have changed `env.CUSTOMER_SDK`_URL to `CUSTOMER_SDK_URL` in the PR.